.maintain/runtime-weight-template.hbs

```diff
@@ -15,70 +17,73 @@
 #![cfg_attr(rustfmt, rustfmt_skip)]
 #![allow(unused_parens)]
 #![allow(unused_imports)]
+#![allow(clippy::as_conversions)]
 
 use frame_support::{traits::Get, weights::Weight};
 use sp_std::marker::PhantomData;
 
 /// Weight functions for `{{pallet}}`.
 pub struct WeightInfo<T>(PhantomData<T>);
-impl<T: frame_system::Config> {{pallet}}::WeightInfo for WeightInfo<T> {
-	{{#each benchmarks as |benchmark|}}
-	{{#each benchmark.comments as |comment|}}
-	/// {{comment}}
-	{{/each}}
-	{{#each benchmark.component_ranges as |range|}}
-	/// The range of component `{{range.name}}` is `[{{range.min}}, {{range.max}}]`.
-	{{/each}}
-	fn {{benchmark.name~}}
-	(
-		{{~#each benchmark.components as |c| ~}}
-		{{~#if (not c.is_used)}}_{{/if}}{{c.name}}: u32, {{/each~}}
-	) -> Weight {
-		// Proof Size summary in bytes:
-		//  Measured:  `{{benchmark.base_recorded_proof_size}}{{#each benchmark.component_recorded_proof_size as |cp|}} + {{cp.name}} * ({{cp.slope}} ±{{underscore cp.error}}){{/each}}`
-		//  Estimated: `{{benchmark.base_calculated_proof_size}}{{#each benchmark.component_calculated_proof_size as |cp|}} + {{cp.name}} * ({{cp.slope}} ±{{underscore cp.error}}){{/each}}`
-		// Minimum execution time: {{underscore benchmark.min_execution_time}}_000 picoseconds.
-		Weight::from_parts({{underscore benchmark.base_weight}}, 0)
-			.saturating_add(Weight::from_parts(0, {{benchmark.base_calculated_proof_size}}))
-			{{#each benchmark.component_weight as |cw|}}
-			// Standard Error: {{underscore cw.error}}
-			.saturating_add(Weight::from_parts({{underscore cw.slope}}, 0).saturating_mul({{cw.name}}.into()))
-			{{/each}}
-			{{#if (ne benchmark.base_reads "0")}}
-			.saturating_add(T::DbWeight::get().reads({{benchmark.base_reads}}))
-			{{/if}}
-			{{#each benchmark.component_reads as |cr|}}
-			.saturating_add(T::DbWeight::get().reads(({{cr.slope}}_u64).saturating_mul({{cr.name}}.into())))
-			{{/each}}
-			{{#if (ne benchmark.base_writes "0")}}
-			.saturating_add(T::DbWeight::get().writes({{benchmark.base_writes}}))
-			{{/if}}
-			{{#each benchmark.component_writes as |cw|}}
-			.saturating_add(T::DbWeight::get().writes(({{cw.slope}}_u64).saturating_mul({{cw.name}}.into())))
-			{{/each}}
-			{{#each benchmark.component_calculated_proof_size as |cp|}}
-			.saturating_add(Weight::from_parts(0, {{cp.slope}}).saturating_mul({{cp.name}}.into()))
-			{{/each}}
-	}
-	{{/each}}
-}
+		impl<T: frame_system::Config> {{pallet}}::WeightInfo for WeightInfo<T> {
+				{{#each benchmarks as |benchmark|}}
+				{{#each benchmark.comments as |comment|}}
+				/// {{comment}}
+				{{/each}}
+				{{#each benchmark.component_ranges as |range|}}
+				/// The range of component `{{range.name}}` is `[{{range.min}}, {{range.max}}]`.
+				{{/each}}
+				fn {{benchmark.name~}}
+				(
+				{{~#each benchmark.components as |c| ~}}
+				{{~#if (not c.is_used)}}_{{/if}}{{c.name}}: u32, {{/each~}}
```

Comment:
Yeah format-on-save in action. I reverted that and also the lints which I think were not necessary anymore, since I get not lint warning if I removed them from all weight files. https://github.com/KILTprotocol/kilt-node/pull/737/commits/198105ea9033cbf3ae87c503a08b80a9d90bbad6 & https://github.com/KILTprotocol/kilt-node/pull/737/commits/0486c12d4c35eefc8b4ea027bf3a32d554595aed

Reply:
Okay strange. I would have assumed, that `#![allow(unused_parens)]` has still to be present in the `weights.rs` files. If not, it makes no sense to add it to the templates

Reply:
No it did not trigger any lint warnings, so I assumed we changed the template to make it irrelevant.